### ISSUE TITLE
Simplify Listaflow to Improve Engagement

### ISSUE THREAD

**Issue by cassiezamparini** [openedx/wg-coordination#130]
See proposal: https://openedx.atlassian.net/wiki/spaces/COMM/pages/4487872513/Simplify+Listaflow+to+Improve+Engagement

**Comment by antoviaque** [comment]
@feanil During the last meeting of the coordination working group, we set as a goal to have a plan for each of the approved proposals from the summit. It's likely going to be tricky to get things done with Christmas, but what would be your plan & timeline beyond that?

**Comment by cassiezamparini** [comment]
@feanil just making notes here for everyone's benefit...

As part of your proposal you want to pare down the Listaflow questions - please let me know if I can get our OpenCraft team to make changes for you? Let me know when you've finalised the questions based on the comments made on your proposal 😄

**Comment by cassiezamparini** [comment]
OpenCraft is working on making updates to the Listaflow form, ie:

- Reduce notification emails to once per cycle.
- Make it easy to unsubscribe from notification emails.
- Make the all questions optional.
- Pare down the survey to just 3 questions.
  - How did you feel during the last sprint? (Include a 5 point opinion scale, like the 10 point scale we do for OpenCraft. 1 = Challenged; 3 = Okay; 5 = Great)
  - What or who are you proud of? Is there any work you'd like to celebrate?
  - Is there anything you're stuck on or need help with, or any areas you'd like input on?

**Comment by antoviaque** [comment]
@cassiezamparini Thanks for the recap!

Note that we also need to keep the question about the time spent, for organizations which won't be reporting it at the organization level, but will keep the reporting at an individual level - like OpenCraft. :)

**Comment by cassiezamparini** [comment]
@antoviaque Just a question on this, I thought we were leaving it out as Axim keeps track of this? And now that we're really making the "Check-in" optional we might not get a lot of numbers reported. Also the majority of people filling in the survey felt this section wasn't "useful" information. I just want to understand who needs the info as I think it might be a bit skewed based on the amount of people actually filling in the form.

@feanil See Xavier's comment above 👆 Please let me know your thoughts...

**Comment by antoviaque** [comment]
@cassiezamparini See https://openedx.atlassian.net/wiki/spaces/COMM/pages/4487872513/Simplify+Listaflow+to+Improve+Engagement?focusedCommentId=4518150147 for the discussion - it depends on how the organization will be reporting their time. Some will do that reporting at the organization level, but some (like us) might keep using the report for this at an individual level.

(PS: Note that when you edit a post on github, it doesn't send a new notification - so I only saw your ping here when I came back to the post, I didn't get notified for the ping itself)

**Comment by cassiezamparini** [comment]
@antoviaque Ah I'd forgotten about this conversation - so thank you for reminding me. Okay I will ask Fox to keep the question in only for OpenCraft. I see the majority of OpenCraft tend to fill it in, would you like this to be mandatory for the team from a reporting perspective?

![Image](https://github.com/user-attachments/assets/c0d794f8-a783-4114-99f0-602b786d8c16)

Ah thank you for the heads up on GitHub not sending a notification if an edit is made!

**Comment by antoviaque** [comment]
@cassiezamparini Yup, within the team we would still need all core contributors to report this to keep our reported totals to be accurate, so it makes sense to make that question mandatory. 👍

**Comment by antoviaque** [comment]
@cassiezamparini How is this task going? Anything blocking?

**Comment by cassiezamparini** [comment]
@antoviaque This task has been completed. However, the changes will not take place until the checklist after this coming one is generated. Right now there's an upcoming one that already has been generated. The first simplified checklist will be available in April :)
